syft/pkg/license_set.go

```diff
@@ -21,77 +22,43 @@ func NewLicenseSet(licenses ...License) (s LicenseSet) {
 	return s
 }
 
-func (s *LicenseSet) get(license License) (uint64, int, *License, error) {
-	id, err := license.Hash()
+func (s *LicenseSet) get(license License) (id artifact.ID, merged bool, err error) {
```

Comment:
This is no longer just fetching an item, it's actively merging it. A name change is warranted. 
```suggestion
func (s *LicenseSet) addToExisting(license License) (id artifact.ID, merged bool, err error) {
```